[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_sync_nodes_are_relay with id: 2024-11-07_09-32-09__63fcdbba-f1cb-4914-bb78-aac525eb383b
[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: common_setup
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node1_2024-11-07_09-32-09__63fcdbba-f1cb-4914-bb78-aac525eb383b__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node2_2024-11-07_09-32-09__63fcdbba-f1cb-4914-bb78-aac525eb383b__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node3_2024-11-07_09-32-09__63fcdbba-f1cb-4914-bb78-aac525eb383b__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.steps.store:store.py:31 Running fixture setup: store_setup
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.217.179
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['53045', '53046', '53047', '53048', '53049']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:436 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:165 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:167 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 53045:53045 -p 53046:53046 -p 53047:53047 -p 53048:53048 -p 53049:53049 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=53047 --rest-port=53045 --tcp-port=53046 --discv5-udp-port=53048 --rest-address=0.0.0.0 --nat=extip:172.18.217.179 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=f645dea7cfd054ecb58888c8095b49f181bda9ba778db9d4a7747ce09bb645e8 --max-connections=1000 --filter-subscription-timeout=600 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=53049 --metrics-logging=true --store=true --relay=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.217.179 waku 10c72d2ddbf2f2eb433aa14f1bc54f13c7f5461eac8b9c0b31fbe15badcee714
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 10c72d2ddbf2. Setting up logs at ./log/docker/node1_2024-11-07_09-32-09__63fcdbba-f1cb-4914-bb78-aac525eb383b__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:179 Started container from image wakuorg/nwaku:latest. REST: 53045
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container b5a8ac711ced. Exiting log stream.
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:53045/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:279 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:53045/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.217.179/tcp/53046/p2p/16Uiu2HAmS312vzF3XRZFQhk4gkJK8T8Y4eGXNTUMwr4JYJk6GhkY","/ip4/172.18.217.179/tcp/53047/ws/p2p/16Uiu2HAmS312vzF3XRZFQhk4gkJK8T8Y4eGXNTUMwr4JYJk6GhkY"],"enrUri":"enr:-L24QIsxxhNQEJjaXv0_mRQr3RmoVmtlqE2HKDbsV2WNWAvbGJfuKzEkMnieHLvao81MGQpcCDgEgZUnAcm1-f04GKECgmlkgnY0gmlwhKwS2bOKbXVsdGlhZGRyc5YACASsEtmzBs82AAoErBLZswbPN90DgnJzhQADAQAAiXNlY3AyNTZrMaEDxst7DCAXjWVbVLehbyUnCEPRR7Bw1hjfpOVj_0moZcWDdGNwgs82g3VkcILPOIV3YWt1MgM"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:284 REST service is ready !!
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.96.205
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['35825', '35826', '35827', '35828', '35829']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:436 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:165 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:167 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 35825:35825 -p 35826:35826 -p 35827:35827 -p 35828:35828 -p 35829:35829 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=35827 --rest-port=35825 --tcp-port=35826 --discv5-udp-port=35828 --rest-address=0.0.0.0 --nat=extip:172.18.96.205 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=cc455ca75b481e2747cf6df41751b0aca8f51fbfb1ec8b802dca1eca8ed2caba --max-connections=1000 --filter-subscription-timeout=600 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=35829 --metrics-logging=true --store=false --store-sync=true --relay=true --discv5-bootstrap-node=enr:-L24QIsxxhNQEJjaXv0_mRQr3RmoVmtlqE2HKDbsV2WNWAvbGJfuKzEkMnieHLvao81MGQpcCDgEgZUnAcm1-f04GKECgmlkgnY0gmlwhKwS2bOKbXVsdGlhZGRyc5YACASsEtmzBs82AAoErBLZswbPN90DgnJzhQADAQAAiXNlY3AyNTZrMaEDxst7DCAXjWVbVLehbyUnCEPRR7Bw1hjfpOVj_0moZcWDdGNwgs82g3VkcILPOIV3YWt1MgM
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.96.205 waku dc768f17a6916524e853e38ea9a87958061441f517290f3bb1b328b60f625713
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID dc768f17a691. Setting up logs at ./log/docker/node2_2024-11-07_09-32-09__63fcdbba-f1cb-4914-bb78-aac525eb383b__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:179 Started container from image wakuorg/nwaku:latest. REST: 35825
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:35825/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:279 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:35825/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.96.205/tcp/35826/p2p/16Uiu2HAkwp5bmUnej2Nozgs6TTKD87N1aVN9zc9A6SL3WPzPQci6","/ip4/172.18.96.205/tcp/35827/ws/p2p/16Uiu2HAkwp5bmUnej2Nozgs6TTKD87N1aVN9zc9A6SL3WPzPQci6"],"enrUri":"enr:-L24QNHXE-uvmXwTXu8Y7Yl2Vvv2_Xx7_GUnPsIgh-gG5Jfqblvf9vE6rL5nBN1RCrvuaLsIyK0FhOg7ylM3zOW6Oh0CgmlkgnY0gmlwhKwSYM2KbXVsdGlhZGRyc5YACASsEmDNBovyAAoErBJgzQaL890DgnJzhQADAQAAiXNlY3AyNTZrMaECI3Tb8R6UvMcbsnQm_A0fPQB9vnQebCGcNNo_So_PpNODdGNwgovyg3VkcIKL9IV3YWt1MhE"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:284 REST service is ready !!
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.183.247
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['51398', '51399', '51400', '51401', '51402']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:436 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:165 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:167 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 51398:51398 -p 51399:51399 -p 51400:51400 -p 51401:51401 -p 51402:51402 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=51400 --rest-port=51398 --tcp-port=51399 --discv5-udp-port=51401 --rest-address=0.0.0.0 --nat=extip:172.18.183.247 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=7dc1be5ae5240bd11296e8c790b27623a585bd9dc7579dd2eb076ac16a3d0dd8 --max-connections=1000 --filter-subscription-timeout=600 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=51402 --metrics-logging=true --store=false --store-sync=true --relay=true --discv5-bootstrap-node=enr:-L24QNHXE-uvmXwTXu8Y7Yl2Vvv2_Xx7_GUnPsIgh-gG5Jfqblvf9vE6rL5nBN1RCrvuaLsIyK0FhOg7ylM3zOW6Oh0CgmlkgnY0gmlwhKwSYM2KbXVsdGlhZGRyc5YACASsEmDNBovyAAoErBJgzQaL890DgnJzhQADAQAAiXNlY3AyNTZrMaECI3Tb8R6UvMcbsnQm_A0fPQB9vnQebCGcNNo_So_PpNODdGNwgovyg3VkcIKL9IV3YWt1MhE
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.183.247 waku 36317847dc002f6f704a8d5578b40fe43320f2af39475e9ba0cab5edc24508cc
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 36317847dc00. Setting up logs at ./log/docker/node3_2024-11-07_09-32-09__63fcdbba-f1cb-4914-bb78-aac525eb383b__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:179 Started container from image wakuorg/nwaku:latest. REST: 51398
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:51398/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:279 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:51398/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.183.247/tcp/51399/p2p/16Uiu2HAmSQx4exCcj8U9DFPtxkxzN7Eguvw7DzArCcf4aNDVkLDm","/ip4/172.18.183.247/tcp/51400/ws/p2p/16Uiu2HAmSQx4exCcj8U9DFPtxkxzN7Eguvw7DzArCcf4aNDVkLDm"],"enrUri":"enr:-L24QEe6zS9tujFQ0tQhSEA_rkE3flZgNj3_zT-ra7-8lN5LInRcRXLO_NcLe65kJ8yuF7FtldWbexcm6oKoIBv3sc8CgmlkgnY0gmlwhKwSt_eKbXVsdGlhZGRyc5YACASsErf3BsjHAAoErBK39wbIyN0DgnJzhQADAQAAiXNlY3AyNTZrMaEDzGrmYqQ-OQGM3I4HjyMOLIlj9ReYN4g0mMPFFzzio0iDdGNwgsjHg3VkcILIyYV3YWt1MhE"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:284 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:35825/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.217.179/tcp/53046/p2p/16Uiu2HAmS312vzF3XRZFQhk4gkJK8T8Y4eGXNTUMwr4JYJk6GhkY"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:51398/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.96.205/tcp/35826/p2p/16Uiu2HAkwp5bmUnej2Nozgs6TTKD87N1aVN9zc9A6SL3WPzPQci6"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:53045/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:35825/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:51398/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:53045/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d '{"payload": "U3RvcmUgd29ya3MhIQ==", "contentTopic": "/myapp/1/latest/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:53045/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d '{"payload": "U3RvcmUgd29ya3MhIQ==", "contentTopic": "/myapp/1/latest/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:53045/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d '{"payload": "U3RvcmUgd29ya3MhIQ==", "contentTopic": "/myapp/1/latest/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:53045/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d '{"payload": "U3RvcmUgd29ya3MhIQ==", "contentTopic": "/myapp/1/latest/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:53045/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d '{"payload": "U3RvcmUgd29ya3MhIQ==", "contentTopic": "/myapp/1/latest/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:53045/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d '{"payload": "U3RvcmUgd29ya3MhIQ==", "contentTopic": "/myapp/1/latest/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:53045/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d '{"payload": "U3RvcmUgd29ya3MhIQ==", "contentTopic": "/myapp/1/latest/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:53045/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d '{"payload": "U3RvcmUgd29ya3MhIQ==", "contentTopic": "/myapp/1/latest/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:53045/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d '{"payload": "U3RvcmUgd29ya3MhIQ==", "contentTopic": "/myapp/1/latest/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:53045/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d '{"payload": "U3RvcmUgd29ya3MhIQ==", "contentTopic": "/myapp/1/latest/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 2 seconds
[35mDEBUG   [0m src.steps.store:store.py:218 Checking that peer wakuorg/nwaku:latest can find the stored messages
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:53045/store/v3/messages?pubsubTopic=%2Fwaku%2F2%2Frs%2F3%2F0&pageSize=100&ascending=true" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"","statusCode":200,"statusDesc":"OK","messages":[{"messageHash":"gZRWv0zobgWiRHkDru5nYdr3Fb1sTOs4QvCFyl0wvtA="},{"messageHash":"EmlLovz4+5yJL8C6EeQP+tKh+NlaCYuoe+CXwCFtW2g="},{"messageHash":"pE6P7yfRrDdm3juFyAIkF023R/CminUSe1mKyFtylDs="},{"messageHash":"yUR+og8+63I6Z+xgnLIF9TQAv7AEJD0nhj6bUrvv/PE="},{"messageHash":"mqLu3D/ZRyTkH6/3gzAGb/hoVGkMX9Mb0sPO54dFC1k="},{"messageHash":"z1OO61kxe9JNuCV9ff8r1imdVPnbnDV2tHkFTcd1+QE="},{"messageHash":"gP0J5KDvAV/wJJI8SlY/SAzFCN3fbsMqveD1byxesJI="},{"messageHash":"kkqTjEJD7nFNHJea9qA0oHkjwoOcgkQ3dgZzob1iNCU="},{"messageHash":"A0w0Ny3dmSWmanmH4yoKTkh84u89DQONMD+4mIKYdUU="},{"messageHash":"c2pmXNjwzrXie9HCK4YLTsRh2IYInBuDqZvpqf2YSFw="}]}'
[35mDEBUG   [0m src.steps.store:store.py:218 Checking that peer wakuorg/nwaku:latest can find the stored messages
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:35825/store/v3/messages?pubsubTopic=%2Fwaku%2F2%2Frs%2F3%2F0&pageSize=100&ascending=true" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"fe4a46609cd4fd591043","statusCode":200,"statusDesc":"OK","messages":[{"messageHash":"gZRWv0zobgWiRHkDru5nYdr3Fb1sTOs4QvCFyl0wvtA="},{"messageHash":"EmlLovz4+5yJL8C6EeQP+tKh+NlaCYuoe+CXwCFtW2g="},{"messageHash":"pE6P7yfRrDdm3juFyAIkF023R/CminUSe1mKyFtylDs="},{"messageHash":"yUR+og8+63I6Z+xgnLIF9TQAv7AEJD0nhj6bUrvv/PE="},{"messageHash":"mqLu3D/ZRyTkH6/3gzAGb/hoVGkMX9Mb0sPO54dFC1k="},{"messageHash":"z1OO61kxe9JNuCV9ff8r1imdVPnbnDV2tHkFTcd1+QE="},{"messageHash":"gP0J5KDvAV/wJJI8SlY/SAzFCN3fbsMqveD1byxesJI="},{"messageHash":"kkqTjEJD7nFNHJea9qA0oHkjwoOcgkQ3dgZzob1iNCU="},{"messageHash":"A0w0Ny3dmSWmanmH4yoKTkh84u89DQONMD+4mIKYdUU="},{"messageHash":"c2pmXNjwzrXie9HCK4YLTsRh2IYInBuDqZvpqf2YSFw="}]}'
[35mDEBUG   [0m src.steps.store:store.py:218 Checking that peer wakuorg/nwaku:latest can find the stored messages
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:51398/store/v3/messages?pubsubTopic=%2Fwaku%2F2%2Frs%2F3%2F0&pageSize=100&ascending=true" -H "Content-Type: application/json" -d 'None'
[31m[1mERROR   [0m src.node.api_clients.base_client:base_client.py:16 HTTP error occurred: 412 Client Error: Precondition Failed for url: http://127.0.0.1:51398/store/v3/messages?pubsubTopic=%2Fwaku%2F2%2Frs%2F3%2F0&pageSize=100&ascending=true. Response content: b'No suitable service peer & none discovered'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Stopping container with id 10c72d2ddbf2
[35mDEBUG   [0m src.node.waku_node:waku_node.py:230 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Stopping container with id dc768f17a691
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container 10c72d2ddbf2. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:230 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Stopping container with id 36317847dc00
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container dc768f17a691. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:230 Container stopped.
[35mDEBUG   [0m tests.conftest:conftest.py:98 Running fixture teardown: check_waku_log_errors
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m tests.conftest:conftest.py:73 Running fixture teardown: attach_logs_on_fail
[35mDEBUG   [0m tests.conftest:conftest.py:74 Test failed, attempting to attach logs to the allure reports
[35mDEBUG   [0m src.libs.common:common.py:30 Attaching file ./log/docker/node3_2024-11-07_09-32-09__63fcdbba-f1cb-4914-bb78-aac525eb383b__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.libs.common:common.py:30 Attaching file ./log/docker/node2_2024-11-07_09-32-09__63fcdbba-f1cb-4914-bb78-aac525eb383b__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.libs.common:common.py:30 Attaching file ./log/docker/node1_2024-11-07_09-32-09__63fcdbba-f1cb-4914-bb78-aac525eb383b__wakuorg_nwaku:latest.log